packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -214,7 +542,32 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
 
   // async isModuleEnabled(moduleName: string): boolean {
 
-  // async enableModule(moduleName: string): Promise<UserOperationResponse> {
+  async enableModule(moduleAddress: string): Promise<UserOpResponse> {
+    const accountContract = await this._getAccountContract()
+    const populatedTransaction = await accountContract.populateTransaction.enableModule(
+      moduleAddress
+    )
+    const tx: Transaction = {
+      to: await this.getAccountAddress(),
+      value: '0',
+      data: populatedTransaction.data as string
+    }
```

Comment:
Call `getEnableModuleData` to get the tx data. Same thing is being done in `getEnableModuleData` method.

Reply:
I see. done